uber/templates/regform.html

```diff
@@ -285,10 +285,36 @@
 <div class="form-group">
     <label for="email" class="col-sm-2 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address" id="mailValue1">
     </div>
 </div>
 
+<div class="form-group">
+    <label for="email" class="col-sm-2 control-label">Email Address Confirmation</label>
+    <div class="col-sm-6">
+        <input type="email" name="email2" id="email2" value=" emailValue2" class="form-control" placeholder="Email Address 2" id="mailValue2">
+    </div>
+</div>
+
+<div hidden class="email-error">
+        <br>
+    <p style="text-indent: 23em; color: red"><b>THE EMAIL Addresses entered do not match, please enter them again.</b></p>
+</div> <br>
+<div hidden class="email-identical">
+        <br>
+    <p style="text-indent: 23em; color: green"><b>THE EMAIL Addresses entered match.</b></p>
+</div> <br>
+
+<script>
+    var emailValue2;
+        if(document.getElementById("mailValue2").value == document.getElementById("mailValue1").value){
+            document.getElementsByClassName("email-identical").removeAttribute("hidden");
+        }
+        else{
+            document.getElementsByClassName("email-error").removeAttribute("hidden");
+        }
+</script>
```

Comment:
Would this even run at all? JS is not my strong suit but I think you need to encapsulate code in a document ready block. We want it to run both when the page is loaded (in case they get a different error AND their email address doesn't match) and whenever they unfocus either field (I hate it when forms tell me the email 'doesn't match' _as I'm typing_, that's dumb).

And yes, jQuery!!!